test/GenerisTestCase.php

```diff
@@ -22,25 +22,18 @@
 namespace oat\generis\test;
 
 use oat\oatbox\filesystem\FileSystemService;
+use League\Flysystem\Memory\MemoryAdapter;
 
 class GenerisTestCase extends TestCase
 {
     use OntologyMockTrait;
 
-    /**
-     * @return FileSystemService
-     */
-    protected function getFileSystemMock($dirs = [])
+    protected function getFileSystemMock($dirs = []): FileSystemService
     {
         $adapterparam = [
-            'testfs' => class_exists('League\Flysystem\Memory\MemoryAdapter')
-                ? [
-                    'class' => \League\Flysystem\Memory\MemoryAdapter::class
-                ]
-                : [
-                    'class' => FileSystemService::FLYSYSTEM_LOCAL_ADAPTER,
-                    'options' => ['root' => \tao_helpers_File::createTempDir()]
-                ]
+            'testfs' => [
+                'class' => MemoryAdapter::class
```

Comment:
Since we have memory adapter as a dependency of generis we could remove check that this class exists in other places as well:
https://github.com/oat-sa/generis/blob/master/test/integration/common/filesystem/DirectoryFilesystemTest.php#L100
https://github.com/oat-sa/generis/blob/master/test/integration/helpers/FileSerializerMigrationHelperTest.php#L180

Reply:
done